claude.md

```diff
@@ -0,0 +1,35 @@
+# Quarto Development Guide
+
+## Project Overview
+
+Quarto is an open-source scientific and technical publishing system built on [Pandoc](https://pandoc.org). This repository contains the source code for various parts of the Quarto ecosystem, with the main CLI implementation housed in a separate repository ([quarto-cli](https://github.com/quarto-dev/quarto-cli)).
+
+### Main Components
+
+- **VS Code Extension**: The primary VS Code extension for working with Quarto documents
+- **Writer**: A web-based editor for Quarto documents
+- **LSP**: Language server for Quarto documents
+- **Core Packages**: Shared libraries used across multiple components
+
+## Repository Structure
+
+The repository is organized as a monorepo using Yarn workspaces:
```

Comment:
I would mention [turbo](https://turborepo.com/) because, AIUI, it changes where node_modules go and how monorepo dependencies are declared in package.json files.